### ISSUE TITLE
lang-html css declaration

### ISSUE THREAD

**Issue by franciscotv** [egoist/hack#14]
I am hesitating about this one but, here it goes.
When downloading the repo directly, the class="lang-html" is not working, so <code class="lang-html"> element doesn't show correctly and any html code isn't showing syntax high-lighting.
I cannot find the lang*, lang-html css declaration anywhere in the repo.
Where is it located? For some reason it works fine in http://hackcss.com/ (even dark mode) but not in a local html file using the downloaded repo.
Thanks in advance.

Regards.

**Comment by egoist** [comment]
did you run `npm run dev` ? the website is generated from src/html/*.jade

**Comment by franciscotv** [comment]
Thanks egoist.
npm and gulp have some troubles in my box, so I am manually connecting all, using git clone and posing the css files in respective folders. All goes well except `class="lang-html"` still, hence the html build is failing.

From the dark and index .jade files i can see that this class="lang-html" gets build by:

      :marked
        ```html
        <link rel="stylesheet" href="/path/to/hack.css" />
         ...
        </body>
        ```

```
:marked
```html
<!-- required in all modes -->
```

code, and that is where the pretty syntax highlighting fails.

Thanks again.

**Comment by franciscotv** [comment]
here is a screenshot of the issue running "npm run dev"
![npm-run-dev-fail](https://cloud.githubusercontent.com/assets/15218851/16937666/a173d54a-4d36-11e6-8478-63bfa7ea69f3.JPG)

**Comment by egoist** [comment]
oh please upgrade your node to at least v4

**Comment by franciscotv** [comment]
thanks egoist, I upgraded my node and git w/o luck for getting the hack theme properly working, although now I have it working fine, kind of. What I had to do was to pull somehow the frameworks-eXXXX.css stylesheet you use. And locate it in my box, then just including it as a <link > in my <head>
I am still wondering how that .css stylesheet gets in your generated .html's
I also found that you use prism as css stylesheet and I am playing with that as well.
Thanks again.
